sr_port_cm/gtcmtr_reversequery.c

```diff
@@ -0,0 +1,99 @@
+/****************************************************************
+ *								*
+ *	Copyright 2001, 2013 Fidelity Information Services, Inc	*
+ *								*
+ * Copyright (c) 2017 YottaDB LLC. and/or its subsidiaries.	*
+ * All rights reserved.						*
+ *								*
+ *	This source code contains the intellectual property	*
+ *	of its copyright holder(s), and is made available	*
+ *	under a license.  If you do not know the terms of	*
+ *	the license, please stop and do not read further.	*
+ *								*
+ ****************************************************************/
+
+#include "mdef.h"
+
+#include <stddef.h>
+
+#include "gtm_string.h"
+
+#include "copy.h"
+#include "gdsroot.h"
+#include "gtm_facility.h"
+#include "fileinfo.h"
+#include "gdsbt.h"
+#include "gdsfhead.h"
+#include "cmidef.h"
+#include "hashtab_mname.h"	/* needed for cmmdef.h */
+#include "cmmdef.h"
+#include "gtcm_find_region.h"
+#include "gtcm_bind_name.h"
+#include "gtcmtr_protos.h"
+#include "gv_xform_key.h"
+#include "gvcst_protos.h"	/* for gvcst_reversequery prototype */
+
+GBLREF connection_struct *curr_entry;
+GBLREF gv_namehead	*gv_target;
+GBLREF gv_key		*gv_currkey;
+GBLREF gv_key		*gv_altkey;
+GBLREF gd_region        *gv_cur_region;
+
+/* Note: This function is very similar to "gtcmtr_query" */
```

Comment:
Needs block comment. Being told to look elsewhere isn't really what's needed.

Reply:
Done.